diff --git a/src/main/java/com/sakurapuare/boatmanagement/pojo/dto/base/BaseDocksDTO.java b/src/main/java/com/sakurapuare/boatmanagement/pojo/dto/base/BaseDocksDTO.java
index e284632..5cee7e1 100644
--- a/src/main/java/com/sakurapuare/boatmanagement/pojo/dto/base/BaseDocksDTO.java
+++ b/src/main/java/com/sakurapuare/boatmanagement/pojo/dto/base/BaseDocksDTO.java
@@ -5,6 +5,9 @@ import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+
+import java.util.List;
+
 @Data
 @ApiModel("码头表")
 @EqualsAndHashCode(callSuper = true)
