diff --git a/src/main/java/com/sakurapuare/boatmanagement/pojo/entity/Docks.java b/src/main/java/com/sakurapuare/boatmanagement/pojo/entity/Docks.java
index b1156a0..1354c56 100644
--- a/src/main/java/com/sakurapuare/boatmanagement/pojo/entity/Docks.java
+++ b/src/main/java/com/sakurapuare/boatmanagement/pojo/entity/Docks.java
@@ -9,6 +9,7 @@ import com.sakurapuare.boatmanagement.pojo.entity.BaseEntity;
 import java.io.Serializable;
 
 import java.io.Serial;
+import java.util.List;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
